### ISSUE TITLE
New statement but no delimiter and unexpected token with REPLACE

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#12020]
As reported at http://stackoverflow.com/questions/35568587/phpmyadmin-4-5-4-unexpected-token-error-in-replace-statement

```
SELECT  *
FROM mytable
WHERE REPLACE (Col, 'a', '')  = 'bbb'
```

This appears to be valid SQL, but phpMyAdmin gives errors based on the delimiter and token problems.

**Comment by udan11** [comment]
I could not reproduce this using the last version of sql-parser.

**Comment by ibennetch** [comment]
Also reported at http://stackoverflow.com/q/35608945/2385479 for version 4.5.1, but was already fixed by at least 4.5.5.1 (maybe earlier, I didn't test extensively).
